server/openejb-cxf-transport/src/main/java/org/apache/openejb/server/cxf/transport/util/CxfUtil.java

```diff
@@ -315,7 +315,6 @@ public static void configureBus() {
                 final InstrumentationManagerImpl manager = InstrumentationManagerImpl.class.cast(mgr);
                 manager.setEnabled(true);
                 manager.setServer(LocalMBeanServer.get());
-                manager.setDaemon(true);
```

Comment:
Does this call no longer exist in the latest CXF version?

Reply:
I guess it was removed in https://github.com/apache/cxf/commit/1cf4fed546904a4a2560f53a2a2391d834b4026c#diff-06b277ed5b1de60d05718de5e55b47a9b0dcaf25360df46fbdaf1978fe70c30f